### ISSUE TITLE
Is it possible for the GHA bot to post and edit a single comment with benchmark results?

### ISSUE THREAD

**Issue by IndrajeetPatil** [lorenzwalthert/touchstone#119]
Instead of posting a new comment for each commit, is it possible for the github-actions bot to edit the existing comment?

The issue is that if a PR has 25 commits, there will be 25 comments from touchstone workflow. This makes it difficult for both contributors and reviewers to navigate the PR and increases their workload by having to hide outdated comments so that the relevant comments can pop out.

This is what codecov-commenter does when it comments on PR with code coverage reports; it just edits the comments in place for each new commit. One can even look at these edits to see how the code coverage changed across commits.

**Comment by lorenzwalthert** [comment]
I don't know if possible. If you can make it work, I am open for PRs, but I won't put significantly more work into {touchstone} anymore.

**Comment by assignUser** [comment]
Yes that is possible https://docs.github.com/en/rest/issues/comments#update-an-issue-comment
I will take a look (no promise when though).
